src/table/components/TableWrapper.jsx

```diff
@@ -15,33 +15,30 @@ import { updateFocus, handleResetFocus, handleFocusoutEvent } from '../utils/han
 import { handleScroll, handleNavigateTop } from '../utils/handle-scroll';
 import announcementFactory from '../utils/announcement-factory';
 
-const useStyles = makeStyles({
-  paper: {
-    height: '100%',
-    backgroundColor: 'rgb(255, 255, 255)',
-    direction: ({ rtl }) => (rtl ? 'rtl' : 'ltr'),
-  },
-  containerOverflowAuto: {
-    height: 'calc(100% - 52px)',
-    overflow: 'auto',
-  },
-  containerOverflowHidden: {
-    height: '100%',
-    overflow: 'hidden',
-  },
-  tablePaginationSection: {
-    backgroundColor: 'rgb(255, 255, 255)',
-    display: 'flex',
-    justifyContent: 'flex-end',
-  },
-  paginationHidden: {
-    display: 'none',
-  },
+const TablePaper = styled(Paper)({
+  height: '100%',
+  backgroundColor: 'rgb(255, 255, 255)',
 });
 
+const TablePaginationSection = styled(Paper)({
+  backgroundColor: 'rgb(255, 255, 255)',
+  display: 'flex',
+  justifyContent: 'flex-end',
+});
+
+const TableContainerSection = styled(TableContainer)(({ containerMode }) => ({
+  height: containerMode ? '100%' : 'calc(100% - 52px)',
+  overflow: containerMode ? 'hidden' : 'auto',
+}));
+
+const CustomTablePagination = styled(TablePagination)(({ paginationHidden }) => ({
+  display: paginationHidden && 'none',
```

Comment:
does this update when going back and forth between analysis/edit?

Reply:
yes, it does. I think that we have a unit test covering it. I gonna double check.